prow/plugins/override/override.go

```diff
@@ -147,20 +155,25 @@ func handle(oc overrideClient, log *logrus.Entry, e *github.GenericCommentEvent)
 
 	mat := overrideRe.FindAllStringSubmatch(e.Body, -1)
 	if len(mat) == 0 {
+		// no /override commands given in the comment
 		return nil
 	}
 
-	overrides := sets.String{}
-
-	for _, m := range mat {
-		overrides.Insert(m[1])
-	}
-
 	org := e.Repo.Owner.Login
 	repo := e.Repo.Name
 	number := e.Number
 	user := e.User.Login
 
+	overrides := sets.String{}
```

Comment:
nit: please use `var overrides sets.String` when you're setting a default value

Reply:
Oops, this should have been `NewString()` (I think they want us to call the constructor?)